src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServerHudsonTest.java

```diff
@@ -150,14 +148,16 @@ public void testTriggeringFromMultipleGerritServers() throws Exception {
         TestUtils.waitForBuilds(projectTwo, 1);
 
         FreeStyleBuild buildOne = projectOne.getLastCompletedBuild();
-        assertSame(Result.SUCCESS, buildOne.getResult());
-        assertEquals(1, projectOne.getLastCompletedBuild().getNumber());
-        assertSame(gerritServerOneName, buildOne.getCause(GerritCause.class).getEvent().getProvider().getName());
+        assertThat(buildOne.getResult()).isEqualTo(Result.SUCCESS);
```

Comment:
IIRC ```Result``` isn't an enum, but a set of singeltons, so checking reference equals is more correct than instance equals IMHO.